### DISCUSSION TITLE
How to zoom in and out without using the Ctrl or Strg keys

### DISCUSSION THREAD

**Discussion by SebyOnGitHub** [jgraph/drawio#4597] [Q&A] [answered]
Hi, we are trying to switch from Miro to Drawio because we noticed that Drawio has better performance.
But we cannot find the settings how to change this so that we dont have to use the ctrl or strg key for zooming with our with the mouse scroolweel which is quiet annoying. (zooming with the mouse wheel only)

Regards

**Comment by davidjgraph** [comment]
see below

**Reply by alderg** [reply]
Try zoomWheel option https://www.drawio.com/doc/faq/configure-diagram-editor

**Reply by SebyOnGitHub** [reply, marked as answer]
Thanks, that works. For anyone else looking for a quick fix:
- Go to the Extras
- Open the Configuration
- Paste in the following:
{
  "zoomWheel": true
}
